excercises/transformation/README.md

```diff
@@ -0,0 +1,34 @@
+<h2 align=center>Week 03</h2>
+
+<h1 align=center>Exercise: <em>Transformations</em></h1>
+
+### Instructions
+
+In this exercise, we will implement transformations on an object; in this case a simple triangle is enough. Our goal is to make our triangle move in the following orbital movement:
+
+![week3-exercise](assets/week3-exercise.gif)
+
+<sub>**Figure 1**: Your goal.</sub>
+
+You can find the starter code [**here**](SDLProject/main.cpp).
+
+The triangle's movement has to satisfy the following:
+1. It must "orbit" the origin (i.e. the centre of the screen).
+2. It must not rotate or scale in any way.
+3. It must orbit in a perfectly circular motion. The radius of its orbit is up to you.
+4. You must only add code in the designated sections, show in the `main.cpp` file as:
+
+```c++
+// ——————————— GLOBAL VARS AND CONSTS FOR TRANSFORMATIONS ——————————— //
+
+// —————————————————————————————————————————————————————————————————— //
+```
+```c++
+// ——————————— YOUR ORBIT TRANSFORMATIONS SHOULD GO HERE ——————————— //
+    
+// ————————————————————————————————————————————————————————————————— //
+```
+
+If you want a hint, you could take a look at [**this**](https://crystalclearmaths.com/wp-content/uploads/2015/11/Unit-Circle.jpg).
+
+[**Solution**](https://www.youtube.com/watch?v=f2bHoTUiMpI)
```

Comment:
As much as this made me laugh, could you find a version that doesn't have the word "p*rn" in the description? lol it could actually get me in trouble.